zephyr/src/supp_main.c

```diff
@@ -205,15 +218,25 @@ static int z_wpas_remove_interface(const char* ifname)
 
 	if (wpa_s->wpa_state != WPA_INTERFACE_DISABLED) {
 		wpa_printf(MSG_ERROR, "Failed to notify remove interface: %s", ifname);
+		send_wifi_state_event(ifname, NET_EVENT_WPA_SUPP_CMD_IFACE_REMOVED,
+		-EINVAL);
```

Comment:
`-EINVAL` only applies to user given inputs.

```suggestion
		-1);
```

Reply:
Done.